MultiServer.py

```diff
@@ -1932,11 +1932,18 @@ async def main(args: argparse.Namespace):
 
     try:
         if not data_filename:
-            import tkinter
-            import tkinter.filedialog
-            root = tkinter.Tk()
-            root.withdraw()
-            data_filename = tkinter.filedialog.askopenfilename(filetypes=(("Multiworld data", "*.archipelago *.zip"),))
+            try:
+                import tkinter
+                import tkinter.filedialog
+            except Exception as e:
+                logging.error("Could not load tkinter, which is likely not installed. "
+                              "This attempt was made because no .archipelago file was provided as argument. "
+                              "Either provide a file or tkinter.")
```

Comment:
```suggestion
                              "Either provide a file or ensure the tkinter module is installed.")
```

Probably a better message than "provide tkinter"

Reply:
well, tkinter can either come as a python package from pip, or on some systems like ubuntu, as an apt package.
Either of these is called package though, not module. So then we might as well call it package.

Reply:
Updated suggestion.